pages/partials/helpLinkPanel.ejs

```diff
@@ -0,0 +1,5 @@
+<div id="linkNav">
+  <a class="btn btn-s btn-secondary" type="button" href="<%=assessment.help_link%>" target="_blank" rel="noopener noreferrer">
```

Comment:
```suggestion
  <a class="btn btn-s btn-secondary" href="<%= assessment.help_link %>" target="_blank" rel="noopener noreferrer">
```

Reply:
Ended up doing this here after inline: [153488d](https://github.com/PrairieLearn/PrairieLearn/pull/6141/commits/153488df3bf3c70f39af2c8821268ce4c7f14d4e)